math.p8

```diff
@@ -1,24 +1,27 @@
-pico-8 cartridge // http://www.pico-8.com
-version 18
-__lua__
--- pico8lib math library
--- by sparr
+--- @module math
+--- Mathematical operations, mostly on numbers
+-- @author by sparr
 
 
-------------------------------------------------------------------------
--- lookup tables
-
--- a^b will overflow when a>power_overflow[b]
 local power_overflow = {32767,181,31,13,7,5,4,3,3,2,2,2,2,2} -- ,1,1,1
```

Comment:
This seems unusable without a comment explaining it. @usage would be even better, but I can do that later.